### PR TITLE
Pre-0.0.54 launch fixes

### PR DESCRIPTION
**Pull request by aduermael**

### CODE REVIEW

File: lua/modules/cubzh.lua [hunk 1]
```diff
@@ -12,14 +12,20 @@ Config = {
 	Items = {},
 }
 
--- local WATER_ALPHA = 220
+Dev.DisplayFPS = true
```
Comment (caillef) [inline lua/modules/cubzh.lua:15]: I think we need to remove this

**Review by caillef — COMMENTED** [review]
LGTM, I think we -need to copy the cobzh version loading objects with shadow to world_editor_common and maybe use only the functions in the world editor common to avoid duplicated code. I know it is specific to the hub but I think this kind of optimisation might be good for all worlds until we found a solution where we can just update the world editor common code to enable all shadows at one point.

File: lua/modules/cubzh.lua (hunk 1 above)
Comment (aduermael) [reply lua/modules/cubzh.lua:15]: @caillef Oh yes, I think we can merge anyway, I'm clearly not done with work in that file.

**Comment by aduermael** [comment]
> LGTM, I think we -need to copy the cobzh version loading objects with shadow to world_editor_common and maybe use only the functions in the world editor common to avoid duplicated code. I know it is specific to the hub but I think this kind of optimisation might be good for all worlds until we found a solution where we can just update the world editor common code to enable all shadows at one point.

Yes, I agree, we'll need to auto set shadows following some rules when loading worlds made with the editor. Let's discuss this, but I think it's not blocking for the release of 0.0.54.

**Review by caillef — APPROVED** [review]

**Pull request merged**